osbs/core.py

```diff
@@ -630,24 +631,29 @@ def wait(self, build_id, states):
         #   1. our object was found and we are returning in the loop
         #   2. our object was not found and we keep waiting (in the loop)
         # Therefore, let's raise here
-        logger.error("build '%s' was not found during wait", build_id)
+        logger.warning("build '%s' was not found during wait", build_id)
         raise OsbsWatchBuildNotFound("build '%s' was not found and response stream ended" %
                                      build_id)
 
     def wait_for_build_to_finish(self, build_id):
-        for retry in range(1, 10):
+        for retry in range(1, 144):
```

Comment:
Probably should be `range(144)` (pre-existing off-by-one).

Reply:
Fixed.